src/templates/common/.github/workflows/Deploy.yml

```diff
@@ -23,7 +24,7 @@ jobs:
     - name: Install Julia
       uses: julia-actions/setup-julia@v1
       with:
-        version: 1.6
```

Comment:
1.6 should have been '1.6' by the way. According to the people who wrote `julia-actions/install-julia`